plugins/CoreConsole/Commands/CoreArchiver.php

```diff
@@ -84,18 +79,6 @@ public static function configureArchiveCommand(ConsoleCommand $command)
             "Forces the value of this option to be used as the URL to Piwik. \nIf your system does not support"
             . " archiving with CLI processes, you may need to set this in order for the archiving HTTP requests to use"
             . " the desired URLs.");
-        $command->addOption('force-all-websites', null, InputOption::VALUE_NONE,
```

Comment:
btw we'll still need to ability to force re-archiving specific websites or all websites or periods etc ideally. Did you remove it because it basically wouldn't be needed if you invalidate reports first?

Reply:
It's just the default now, since there's no queue or need to skip the visits check or anything. Ie, we by default look for every invalidated archive, then the command options are just use to filter out what we don't want to archive right now.

If you want to force anything to be archived that currently has a usable archive then, yes, you have to invalidate it first, then it will get picked up by an archiving process.

Reply:
btw @diosmosis the force-periods, force-idsites, and force-segments will still be needed I would say. We use that actually quite often, eg when debugging issues or when needing tp give certain archives a priority to not needing to wait for potentially hours or days until they would be archived otherwise. Like when other archives before that would be archived and it would take ages until the ones are archived that you want to debug or need right now. Maybe these parameters could be forwarded eg to getNextArchive should any of them be set? Force all websites wouldn't be needed

Reply:
They are all still supported (by `ArchiveFilter` and by the SELECT to get the next archive). We can also move them to the getNextArchive call to issue less queries overall probably.